acceptance/analyzer_test.go

```diff
@@ -395,343 +575,150 @@ func testAnalyzer(t *testing.T, when spec.G, it spec.S) {
-						output := h.DockerRunAndCopy(t,
-							containerName,
-							copyDir,
-							ctrPath("/layers"),
-							analyzeImage,
-							h.WithFlags(
-								"--network", registryNetwork,
-								"--env", "CNB_PLATFORM_API="+latestPlatformAPI,
-							),
-							h.WithArgs(
-								ctrPath(analyzerPath),
-								"-skip-layers",
-								noAuthRegAppImage,
-							),
-						)
+							output := h.DockerRun(t,
+								analyzeImage,
+								h.WithFlags(append(
+									dockerSocketMount,
+									"--volume", cacheVolume+":/cache",
+									"--env", "CNB_PLATFORM_API="+apiString,
+								)...),
+								h.WithBash(
+									fmt.Sprintf("chown -R 9999:3333 /cache; chmod -R 775 /cache; %s -daemon -cache-dir /cache some-image; ls -alR /cache", analyzerPath),
+								),
+							)
 
-						assertAnalyzedMetadata(t, filepath.Join(copyDir, "layers", "analyzed.toml"))
-						assertWritesStoreTomlOnly(t, copyDir, output)
+							h.AssertMatch(t, output, "9999 3333 .+ \\.")
+							h.AssertMatch(t, output, "9999 3333 .+ committed")
+							h.AssertMatch(t, output, "2222 3333 .+ staging")
+						})
 					})
 				})
 			})
 		})
 
-		when("cache is provided", func() {
-			when("cache image case", func() {
+		when("registry case", func() {
+			it("writes analyzed.toml", func() {
+				h.DockerRunAndCopy(t,
+					containerName,
+					copyDir,
+					ctrPath("/layers/analyzed.toml"),
+					analyzeImage,
+					h.WithFlags(
+						"--env", "CNB_PLATFORM_API="+apiString,
+					),
+					h.WithArgs(
+						ctrPath(analyzerPath),
+						"some-image",
+					),
+				)
+
+				assertAnalyzedMetadata(t, filepath.Join(copyDir, "analyzed.toml"))
+			})
+
+			when("app image exists", func() {
 				when("auth registry", func() {
-					var authRegCacheImage, cacheAuthConfig string
+					var authRegAppImage, appAuthConfig string
 
 					it.Before(func() {
-						metadata := minifyMetadata(t, filepath.Join("testdata", "analyzer", "cache_image_metadata.json"), platform.CacheMetadata{})
-						authRegCacheImage, cacheAuthConfig = buildAuthRegistryImage(
+						metadata := minifyMetadata(t, filepath.Join("testdata", "analyzer", "app_image_metadata.json"), platform.LayersMetadata{})
+						authRegAppImage, appAuthConfig = buildAuthRegistryImage(
 							t,
-							"some-cache-image-"+h.RandString(10),
-							filepath.Join("testdata", "analyzer", "cache-image"),
+							"some-app-image-"+h.RandString(10),
+							filepath.Join("testdata", "analyzer", "app-image"),
 							"--build-arg", "fromImage="+containerBaseImage,
 							"--build-arg", "metadata="+metadata,
 						)
 					})
 
 					it.After(func() {
-						h.DockerImageRemove(t, authRegCacheImage)
+						h.DockerImageRemove(t, authRegAppImage)
 					})
 
 					when("registry creds are provided in CNB_REGISTRY_AUTH", func() {
-						it("restores cache metadata", func() {
+						it("restores app metadata", func() {
+							h.SkipIf(t, api.MustParse(apiString).Compare(api.MustParse("0.7")) >= 0, "Platform API >= 0.7")
```

Comment:
```suggestion
							h.SkipIf(t, api.MustParse(apiString).Compare(api.MustParse("0.7")) >= 0, "Platform API >= 0.7 does not read app layer metadata")
```

Reply:
Again when we have https://github.com/buildpacks/lifecycle/issues/572 we'll probably add an assertion that the app is reachable and we can read the mixin data.